chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
 > # 44. The Optimization Verification test
 
---> _replace THIS LINE by your translation for the above line_
-
+# 44. Kiểm tra xác minh tối ưu hóa
 
 > Suppose you are building a speech recognition system. Your system works by inputting an audio clip ​*A*,​ and computing some Score​<sub>A​</sub>(​*S*) for each possible output sentence ​*S​*. For example, you might try to estimate Score​<sub>A​</sub>(​*S*) = P(​*S​*|​*A​*), the probability that the correct output transcription is the sentence ​*S*,​ given that the input audio was *​A*.
 
---> _replace THIS LINE by your translation for the above line_
-
+Giả sử bạn đang xây dựng một hệ thống nhận dạng giọng nói. Hệ thống của bạn hoạt động bằng cách nhập một đoạn âm thanh ​*A*,​ và tính toán một số Điểm​<sub>A​</sub>(​*S*) cho mỗi câu đầu ra có thể tạo ​*S​*. Ví dụ: bạn có thể thử ước tính Điểm​<sub>A​</sub>(​*S*) = P(​*S​*|​*A​*), xác suất đầu ra chính xác là câu ​*S*,​ với âm thanh đầu vào là *​A*.
```

Comment:
* Chỗ "possible output" có thể dịch là "đầu ra khả dĩ", từ "khả dĩ" trong tiếng Việt rất gần với từ "possible".
* Chỗ ", given that ..." tương đương với xác suất có điều kiện trong tiếng Việt. Em có thể dịch là "với điều kiện âm thanh đầu vào là *A*."